packages/firebase_auth/lib/firebase_auth.dart

```diff
@@ -411,6 +411,15 @@ class FirebaseAuth {
     return currentUser;
   }
 
+  /// Sets the user-facing language code for auth operations that can be
+  /// internationalized, such as [sendEmailVerification].
```

Comment:
Please include the formatting requirements (IETF in BCP47) in this comment.

Eg: as in [Android reference docs](https://firebase.google.com/docs/reference/android/com/google/firebase/auth/FirebaseAuth.html#setLanguageCode(java.lang.String))

Reply:
done